composer.json

```diff
@@ -20,7 +20,8 @@
         "orchestra/testbench": "^6.0",
         "phpunit/phpunit": "^9.3",
         "spatie/laravel-ray": "^1.9",
-        "tipoff/support": "^1.2.1",
+        "tipoff/authorization": "^1.0.1",
```

Comment:
This **could** be dangerous.  By including it here, it becomes a require-dev dependency for all other packages, but we all run w/dev packages installed and CI/CD does as well.  So, we could be lulled into thinking all is ok and the package will be there in production installs when it really won't be.